packages/garbo/src/tasks/freeGiantSandworm.ts

```diff
@@ -293,12 +294,28 @@ function hasWorms(): boolean {
   return (_hasWorms ??= expectedFreeGiantSandwormQuestFights() > 0);
 }
 
+const REJECTION = 1 / 10;
+const BASE_RATE = 1 / 100;
+let _sandwormRate: number;
+function sandwormRate(): number {
+  if (!_sandwormRate) {
+    Outfit.from(
+      sandwormSpec(),
+      new Error("Failed to generate Sandworm outfit"),
+    ).dress();
+    _sandwormRate =
+      REJECTION * BASE_RATE * (1 + getModifier("Item Drop") / 100);
```

Comment:
the issue here is the Item Drop modifier is inaccurate until we maximize and buff. actually are we even buffing with the common item drop effects?

Reply:
I don't think so but great question

Reply:
I think this isn't crazy accurate but the question is will this be _more_ accurate than what we're currently getting? it looks like we used to use `0.02 * price of melange` and now use `0.01`, which is causing #2138

Reply:
yeah it's fine we should just go with this for now